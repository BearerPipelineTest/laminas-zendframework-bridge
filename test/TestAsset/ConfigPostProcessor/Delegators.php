<?php

return [
    'dependencies' => [
        'delegators' => [
            'Buzzer' => [
                '\Zend\ServiceManager\Proxy\LazyServiceFactory',
            ],
        ],
    ],
    'service_manager' => [
        'delegators' => [
            'Buzzer' => [
                '\Zend\ServiceManager\Proxy\LazyServiceFactory',
            ],
        ],
    ],
];
