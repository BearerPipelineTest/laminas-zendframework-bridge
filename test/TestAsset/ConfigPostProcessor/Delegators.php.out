<?php

return [
    'dependencies' => [
        'delegators' => [
            'Buzzer' => [
                '\Laminas\ServiceManager\Proxy\LazyServiceFactory',
            ],
        ],
    ],
    'service_manager' => [
        'delegators' => [
            'Buzzer' => [
                '\Laminas\ServiceManager\Proxy\LazyServiceFactory',
            ],
        ],
    ],
];
